repo/datastore.go

```diff
@@ -49,11 +48,29 @@ type ThreadStore interface {
 	DeleteByName(name string) error
 }
 
+type DeviceStore interface {
```

Comment:
devices are stored locally. each time a thread is added, invites are sent to each paired device.